gdctools/gdc_mirror.py

```
@@ -75,8 +81,20 @@ def config_customize(self):
         #      ONLY supports MIRRORING of legacy, nothing else
         api.set_legacy(config.mirror.legacy)
 
+
+        # Check for append in config file, then allow commandline to override it
+        if config.mirror.append:
+            # Append mode has been requested in config file, coerce to boolean
+            # TODO also validate legal false values
+            value = config.mirror.append.lower()
+            config.mirror.append = (value in ["1", "true", "on", "yes"])
```

Should probably throw an error if it's an unknown false value